server/model-config.json

```diff
@@ -14,22 +14,26 @@
     ]
   },
   "User": {
-    "dataSource": "db"
+    "dataSource": "pg"
   },
   "AccessToken": {
-    "dataSource": "db",
+    "dataSource": "pg",
     "public": false
   },
   "ACL": {
-    "dataSource": "db",
+    "dataSource": "pg",
     "public": false
   },
   "RoleMapping": {
-    "dataSource": "db",
+    "dataSource": "pg",
     "public": false
   },
   "Role": {
-    "dataSource": "db",
+    "dataSource": "pg",
     "public": false
+  },
+  "paste": {
```

Comment:
the upper case ones are buildin loopback things I haven't touched, but I'm probably going to make those lowercase since it doesn't seem that you ever new the models - but we will see what we see.

Also yes is a loopback config thing, you can change the models programatically with JS but the model generator give the json and they seem to want you to use the JS only to override methods